jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/execution/sam/SamVersionTest.kt

```diff
@@ -56,7 +56,6 @@ class SamVersionTest {
     @Test
     fun incompatableSamVersion_tooHigh() {
         val versions = arrayOf(
-                "0.11.0",
```

Comment:
Consider adding some kind of `SamCommon.expectedSamMaxVersion  + increment` value here, so it is always testing a relevant value

Reply:
It is non-inclusive so we could just use the version itself